### PR TITLE
Convert Home page to Vue SFC

### PR DESCRIPTION
**Pull request by pirlgon**
currentRole has been removed from base.html, it was only used in home.html.
isAdmin and groups are now passed as props to the InstanceList component.
The groups filter has been fixed.

In the second commit, the `InstanceList` component has been renamed and moved to `Home` view.

### CODE REVIEW

File: ui/temboardui/static/src/components/home/InstanceList.vue [hunk 1]
```diff
@@ -17,16 +18,15 @@ const loading = ref(false);
 const instances = ref([]);
 const search = ref(route.query.q);
 const sort = ref(route.query.sort || "status");
-const groups = ref([]);
-const groupsFilter = ref(route.query.groups ? route.query.groups.split(",") : []);
+const groupsFilter = reactive(route.query.groups ? route.query.groups.split(",") : []);
```
Comment (pgiraud) [inline ui/temboardui/static/src/components/home/InstanceList.vue:21]: Can you please elaborate why you changed from `ref` to `reactive` here?

Comment (pirlgon) [reply ui/temboardui/static/src/components/home/InstanceList.vue:21]: Filter changes are not caught by the watch function used to update the route, in this case the reactive object must be used to correct the problem.

Comment (pgiraud) [reply ui/temboardui/static/src/components/home/InstanceList.vue:21]: Can you please isolate the change to "fix the groupsFilter" into its own commit? And explain what was "fixed".

I think there are other options than using `reactive`.

Comment (bersace) [reply ui/temboardui/static/src/components/home/InstanceList.vue:21]: Ou en commentaire du code, ça évite de blamer et de dépendre de git.

File: ui/temboardui/templates/home.html [hunk 2]
```diff
@@ -8,15 +8,11 @@
...
-  <a class="btn btn-link fullscreen" href data-toggle="popover" data-trigger="hover" data-content="Full screen" data-placement="bottom">
-    <i class="fa fa-expand"></i>
-  </a>
-</div>
-<div id="instances">
+<div id="app">
+  <instance-list
+    :is-admin="{{ 'true' if role.is_admin else 'false' }}"
+    :groups='{% raw json.dumps([i[0] for i in groups]) %}'>
+</instance-list>
```
Comment (pgiraud) [inline ui/temboardui/templates/home.html]: Can you please fix the indentation here?

File: ui/temboardui/static/src/components/home/InstanceList.vue (hunk 1 above)
Comment (pgiraud) [reply ui/temboardui/static/src/components/home/InstanceList.vue]: Thanks for splitting commits and finding an other workaround.

**Review by pgiraud — APPROVED** [review]

**Pull request merged**